utils.py

```diff
@@ -128,3 +129,48 @@ async def safesend(ctx: Context,
                        + ("[Rest of o" if safe else "[O")
                        + "utput too long to send as message. Sorry.]",
                        file=file)
+
+
+async def isDM(message: Message) -> bool:
+    """Helper function to see if this message was a DM"""
+    if not message.guild:
+        return True
+    else:
+        return False
+
+
+async def isEmbed(message: Message) -> bool:
+    """Helper function to see if this message was an Embed"""
+    if message.embeds:
+        return True
+    else:
+        return False
```

Comment:
```suggestion
    return bool(message.embeds)
```
or `return not not message.embeds` (requires no global lookup) if you want to trade efficiency for readability. or just `return message.embeds` and tweak the typing,